R/animate_tour.R

```diff
@@ -28,7 +28,7 @@
 animate_tour <- function(data,
                          cols = where(is.numeric),
                          tour_path = tourr::grand_tour(d = 2),
-                         display = d3tourr::display_scatter(),
+                         display = detourr::display_scatter(),
```

Comment:
```suggestion
                         display = display_scatter(),
```
if the function is from the package itself, no need to use `::`